style.css

```diff
@@ -83,11 +85,42 @@ form {
 }
 
   #messages li { padding: 5px 10px; }
-    
+
     #messages li:nth-child(odd) { background: #eee; }
 
 
 /* == Individual messages == */
-.time { color: #2c3e50; }
+.time { color: #9c9c9c; }
 
-.name { color: #2980b9; }
+.name {
+  color: #2980b9;
+  margin-right: 0.8em;
+}
+
+.name,
```

Comment:
Hooray for the fact that you're using classes and not ID selectors! 🎉 :heart:
https://github.com/dwyl/style-guide#general-points